pkg/apis/batch/types.go

```diff
@@ -18,11 +18,55 @@ package batch
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/api"
 )
 
 // +genclient=true
 
+// QueueJob represents the configuration of a QueueJob.
+type QueueJob struct {
```

Comment:
Agree; maybe we can make "queuejob" small enough to contains more jobs in the queue.